### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.cosmosdb.table.models.AzureBatchValidationError
  fullName: azure.cosmosdb.table.models.AzureBatchValidationError
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - type: builtins.object
        type: builtins.BaseException
      type: builtins.Exception
    type: azure.common.AzureException
  langs:
  - python
  module: azure.cosmosdb.table.models
  name: AzureBatchValidationError
  source:
    id: AzureBatchValidationError
    path: azure-cosmosdb-table\azure\cosmosdb\table\models.py
    remote:
      branch: master
      path: azure-cosmosdb-table\azure\cosmosdb\table\models.py
      repo: https://github.com/Azure/azure-cosmosdb-python.git
    startLine: 29
  type: class
  uid: azure.cosmosdb.table.models.AzureBatchValidationError
- children: []
  class: azure.cosmosdb.table.models.AzureBatchValidationError
  fullName: azure.cosmosdb.table.models.AzureBatchValidationError
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - type: builtins.object
        type: builtins.BaseException
      type: builtins.Exception
    type: azure.common.AzureException
  langs:
  - python
  module: azure.cosmosdb.table.models
  name: AzureBatchValidationError
  source:
    id: AzureBatchValidationError
    path: azure-cosmosdb-table\azure\cosmosdb\table\models.py
    remote:
      branch: master
      path: azure-cosmosdb-table\azure\cosmosdb\table\models.py
      repo: https://github.com/Azure/azure-cosmosdb-python.git
    startLine: 29
  type: class
  uid: azure.cosmosdb.table.models.AzureBatchValidationError
references: []
